tests/test-derivation.nix

```diff
@@ -9,32 +14,51 @@ let
       dontRun ? false,
       ...
     }:
+    let
+      testList =
+        if builtins.isList nvim then
+          nvim
+        else
+          [
+            {
+              derivation = nvim;
+              inherit name dontRun;
+            }
+          ];
```

Comment:
Passing `nvim` as a list feels like a hack, maybe this should be its own `mkTestDerivationFromNvims` function?

Or at least handle it as a separate argument:

```nix
{
  name,
  nvim ? abort "nvim is required if `tests` is not provided.",
  tests ? [ { derivation = nvim; inherit name dontRun; } ],
  dontRun ? false,

Reply:
Yeah I was not of the best way to do that. I would have liked to _force_ to have a list, but because we expose this I wouldn't want to break user configurations.

Reply:
> because we expose this I wouldn't want to break user configurations

Agreed, we should avoid breaking changes